### PR TITLE
Styling changes and centralized help inline docs

### PR DESCRIPTION
**Pull request by bradymiller**

### CODE REVIEW

File: interface/themes/oe-bootstrap.scss [hunk 1]
```diff
@@ -1,4 +1,7 @@
 /*below section is override bootstrap 3 for openemr and some new classes all prefixed with oe-*/
+input[type=checkbox], input[type=radio] {
+	margin: 4px;
+}
```
Comment (bradymiller) [inline interface/themes/oe-bootstrap.scss]: hi @zbig01 and @d3sandoval ,
I am finding it really strange that the above changes are not "taking" in the color css scripts (only are taking in light and manila). Is this meant to happen.
To produce the css scripts, I ran `npm run build` twice

Comment (bradymiller) [reply interface/themes/oe-bootstrap.scss]: Ok, I figured out the mystery :)
oe-bootstrap is simply not included in the colors scss, which I assume is purposeful.

**Comment by zbig01** [comment]
Yes. These rules were the ones that you had added tonyhose two stylesheets in September. They have been placed I a separate scss file by Dani Sandoval

**Comment by zbig01** [comment]
Looks good. Thanks, now we can try to get the reworked scripts into the codebase :)

**Comment by bradymiller** [comment]
Great! bringing it into codebase now :)

**Pull request merged**